packages/core/addon/validators/request-metrics.js

```diff
@@ -1,16 +1,13 @@
 /**
- * Copyright 2017, Yahoo Holdings Inc.
+ * Copyright 2020, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
-
-import { get } from '@ember/object';
 import BaseValidator from 'ember-cp-validations/validators/base';
-import { isEqual } from 'lodash-es';
-import { getRequestMetrics } from 'navi-core/utils/chart-data';
+import { getRequestDimensions } from 'navi-core/utils/chart-data';
 
 export default BaseValidator.extend({
   validate(value, options /*, model, attribute*/) {
-    let requestMetrics = getRequestMetrics(get(options, 'request'));
-    return isEqual(value, requestMetrics);
+    const { request } = options;
+    return getRequestDimensions(request).length === 0;
```

Comment:
What is this validating?

Reply:
Good catch, this isn't really needed anymore since we don't add the metrics to our static config